catboost/python-package/catboost/core.py

```diff
@@ -183,8 +183,10 @@ class EFstrType(Enum):
     Interaction = 3
     """Calculate SHAP Values for every object."""
     ShapValues = 4
+    """Calculate approximate SHAP Values for every object."""
+    ApproximateShapValues = 5
```

Comment:
Please don't change order of old fstr types, add new to the end of the list.

Reply:
Moved it to the end of enumeration